roadside-forms-frontend/frontend_web_app/src/components/Forms/TwentyFourHourForm/testAdministered.js

```diff
@@ -6,7 +6,7 @@ import { DatePickerField } from '../../common/DateField/DatePicker';
 export const TestAdministered = (props) => {
     const { values } = useFormikContext();
     return(
-        <>{values["type-of-prohibition"] !== '' && (<div className='border-design-form left text-font'>
+        <>{values["prescribed-device"] === 'YES' && (<div className='border-design-form left text-font'>
```

Comment:
I would like to if possible and if its simple to move away from using the string YES and NO to using true and false boolean types if we can.

Reply:
I will see if this can be changed as this will need changes on so many places, i would do that with other PR.